### PR TITLE
Stop releasing prescriptions as a container image

### PR DESCRIPTION
**Pull request by fridex**
#### What type of PR is this?

Related: https://github.com/thoth-station/prescriptions/issues/50

### CODE REVIEW

File: .aicoe-ci.yaml [hunk 1]
```diff
@@ -1,14 +1,2 @@
 check:
 - thoth-build
```
Comment (harshad16) [inline .aicoe-ci.yaml:2]:
```suggestion
check: []
```

**Review by harshad16 — CHANGES_REQUESTED** [review]
lgtm
thanks , just one comment

File: .aicoe-ci.yaml (hunk 1 above)
Comment (fridex) [reply .aicoe-ci.yaml:2]: Thanks 👍🏻

**Comment by harshad16** [comment]
@fridex can you please rebase 
i will review once again and merge it

**Comment by fridex** [comment]
> @fridex can you please rebase
> i will review once again and merge it

Thanks, rebased now. 👍🏻

**Comment by harshad16** [comment]
@fridex rebase removed the change i had suggested
can you make the .aicoe-ci.yaml as
```
check: []
```

**Comment by fridex** [comment]
Ah, sorry about that. Now pushed again with the change. PTAL

**Review by harshad16 — APPROVED** [review]
lgtm
thanks :100:

**Pull request merged**